lib/anagram_cases.rb

```diff
@@ -0,0 +1,24 @@
+class AnagramCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub(/[ -]/, '_')
+  end
+
+  def work_load
+    actual = "Anagram.new(\"#{subject}\").match(#{candidates})"
+    "assert_equal #{expected}, #{actual}"
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def show_comment
+    comment.nil? ? '' : "# #{comment}\n\t\t"
```

Comment:
I was enforcing this because I didn't want a blank line in the generated file if there are no comments for the test case. But it makes sense that no formatting should be done here.